spaceship/lib/spaceship/tunes/availability.rb

```diff
@@ -78,7 +74,7 @@ def territories
       end
 
       def b2b_users
-        @b2b_users ||= raw_data['b2bUsers'].map { |user| B2bUser.new(user) }
+        @b2b_users ? @b2b_users : raw_data['b2bUsers'].map { |user| B2bUser.new(user) }
```

Comment:
Can we change this to `@b2b_users || raw_data['b2bUsers'].map { |user| B2bUser.new(user) }`?